domain/model/state/types.go

```diff
@@ -146,11 +146,17 @@ type dbUUID struct {
 	UUID string `db:"uuid"`
 }
 
-// dbModelUUID represents the model uuid from the model table.
-type dbModelUUID struct {
+// dbModelUUIDRef represents the model uuid from the model table. As referenced
+// by other tables in the DDL. Use this type when you are working model uuid's
+// as foreign keys.
+type dbModelUUIDRef struct {
 	ModelUUID string `db:"model_uuid"`
 }
 
+type dbModelUUID struct {
```

Comment:
Existing code, but we've started to drop the `db` prefix from these type structs. I think we started doing it early on in cloud package and here but just about everything else doesn't do it.

Reply:
Everything in this file does it.